html/customer.html

```diff
@@ -39,7 +39,14 @@ <h2 class="header-h2-name">
                         </div>
                         <!-- Search Content -->
                         <div class="search-content-div">
-                          <input type="text" class="search-content-input" name="#" id="#" placeholder="Search E-commerce App">
+                          <input 
+                          type="text" 
+                          class="search-content-input" 
+                          id="search-input" 
+                          placeholder="Search E-commerce App"
+                          title="Type in a product name">
+                          <ul id="myUL">
```

Comment:
I think it would be helpful if this element had a more descriptive id than `myUL`.  How about `searchResults`?

Reply:
done